### PR TITLE
ignore common private directories

### PR DESCRIPTION
**Pull request by schosterbarak**
Ignore node_modules , .terraform and .serverless directories when parsing files

### CODE REVIEW

File: checkov/common/runners/base_runner.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
+import os
+
 from abc import ABC, abstractmethod
 
 from checkov.runner_filter import RunnerFilter
 
+IGNORED_DIRECTORIES_ENV = os.getenv('IGNORED_DIRECTORIES',"node_modules,.terraform,.serverless")
```
Comment (tronxd) [inline checkov/common/runners/base_runner.py:7]: I would add this env variable to docs, explaining it's a comma separated string of ignored directories.

File: checkov/cloudformation/runner.py [hunk 2]
```diff
@@ -35,6 +35,7 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
 
         if root_folder:
             for root, d_names, f_names in os.walk(root_folder):
+                [d_names.remove(d) for d in list(d_names) if d in ignored_directories]
```
Comment (tronxd) [inline checkov/cloudformation/runner.py:38]: I would refactor the filtering logic into checkov.common or to base runner class

File: checkov/common/runners/base_runner.py (hunk 1 above)
Comment (prabhu) [reply checkov/common/runners/base_runner.py:7]: `CK_IGNORED_DIRECTORIES` something with a prefix

**Comment by prabhu** [comment]
lgtm

**Pull request merged**